README.md

```diff
@@ -56,16 +65,16 @@ See .env.sample for example values
   - Alternatively you can clone this repo then remove the git folder to initialize a new repo
 
```

Comment:
I would recommend changing this from saying to fork the repo to import the repository. This will make a new instance of the repo that isn't linked to the original. Github has said in the past that forks are for temporary work that will eventually be merged back into the original project.

https://web.archive.org/web/20190228074506/https://hisaac.net/2016/11/12/why-commits-to-forks-on-github-dont-count-toward-contributions/